lib/webenginesettings.cpp

```diff
@@ -191,20 +204,27 @@ void SailfishOS::WebEngineSettings::initialize()
     qreal touchStartTolerance = dragThreshold / QGuiApplication::primaryScreen()->physicalDotsPerInch();
     engineSettings->setPreference(QString("apz.touch_start_tolerance"), QString("%1f").arg(touchStartTolerance));
 
-    qreal pixelRatio = SAILFISH_WEBENGINE_DEFAULT_PIXEL_RATIO * silicaTheme->pixelRatio();
-    // Round to nearest even rounding factor
-    pixelRatio = qRound(pixelRatio / 0.5) * 0.5;
+    QSize screenResolution = QGuiApplication::primaryScreen()->size();
+    int screenWidth = screenResolution.width();
+
+    qreal pixelRatio = mappedPixelRatio(screenResolution);
```

Comment:
Uh, this mapping of exact resolutions feels a bit hacky approach. I can understand doing such for just getting stuff done quickly out of the way, but in that case I could skip building infrastructure for doing more overrides and just have an extra if branch here. Maybe with some TODO type of comment to think twice before adding more.

Reply:
I'll add a todo. I actually think it other way around. Mapping should be primary and only fallback to something else. This is precise and accurate value rendering good size web content that is well readable whilst not rendering fisher pricy feeling.

Reply:
But it shouldn't be that surprising encountering devices with exactly the same resolution while quite different screen sizes? Mappings stop working at that point?